doc/source/changes/version_0_34.rst.inc

```diff
@@ -67,3 +67,6 @@ Fixes
 ^^^^^
 
 * fixed Array.insert when no label is provided (closes :issue:`879`).
+
+* fixed comparison between Array and None returning False instead of an array of boolean values
```

Comment:
mention Axis too

Reply:
In the same bullet ?
Because the problem with Axis doesn't look like the same with the one with None.

Reply:
It is the same problem AFAIK and you fixed it in the same commit too, but you can explain it in two different bullets I don't really care about that.